rustainers/src/runner/inner.rs

```diff
@@ -287,6 +328,83 @@ pub(crate) trait InnerRunner: Display + Debug + Send + Sync {
         Ok(())
     }
 
+    fn get_docker_host(&self) -> Option<String> {
+        env::var("DOCKER_HOST").ok()
+    }
```

Comment:
This code seams specific to docker "runner"

Reply:
To be honest, we look that this variable is not defined manually. In the cases of podman and docker it's undefined. It can happen in the case of podman and docker running at the same time that you are asked to
`Another process was listening on the default Docker API socket address.
You can still connect Docker API clients by setting DOCKER_HOST using the
following command in your terminal session:`

To be honest it's not clear to me